frontend/src/app/main/data/workspace/transforms.cljs

```diff
@@ -587,16 +588,12 @@
                               [(assoc move-vector :x 0) :x]
 
                               :else
-                              [move-vector nil])
+                              [move-vector nil])]
 
-                            nesting-loop? (some #(cfh/components-nesting-loop? objects (:id %) target-frame) shapes)
-                            is-component-copy? (ctk/in-component-copy? (get objects target-frame))]
```

Comment:
Are these checks done somewhere else? I can't find it.

Reply:
yes, they are implicit in the call to:
`[target-frame _] (ctn/find-valid-parent-and-frame-ids target-frame objects shapes)`